Base/usr/share/man/man2/pipe.md

```diff
@@ -23,8 +23,8 @@ from `pipefd[0]` will return EOF.
 
 ## Examples
 
-The following program creates a pipe, then forks, the child then
-writes some data to the pipe which the parent reads:
+The following program creates a pipe, forks, and then the child
+writes some data to the pipe (which the parent reads):
```

Comment:
I'm fine with the former change (then / and then), but not the latter: the parent reading it is as important as the rest, and shouldn't be parenthesised.